### ISSUE TITLE
ERROR: No video formats found;

### ISSUE THREAD

**Issue by tuannv1104** [ytdl-org/youtube-dl#29017]
## Checklist

- [x] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2021.04.26**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
[debug] System config: []
[debug] User config: ['--abort-on-error', '--no-warnings', '--console-title', '--batch-file=batch-file.txt', '--user-agent', 'Mozilla/5.0 (X11; Linux i686; rv:21.0) Gecko/20100101 Firefox/21.0', '--cookies', 'cookies.txt', '--socket-timeout', '10', '--retries', '10', '--continue', '--fragment-retries', '10', '--min-sleep-interval', '60', '--max-sleep-interval', '300', '--max-downloads', '150', '-f', 'best[height<=720]/worst[height>720]', '-o', '%(playlist_title)s/%(chapter_number)s - %(chapter)s/%(playlist_index)d-%(title)s.%(ext)s']
[debug] Custom config: []
[debug] Command-line args: ['--verbose']
[debug] Batch file urls: ['https://app.pluralsight.com/library/courses/google-cloud-platform-fundamentals-core-infrastructure/table-of-contents']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2021.04.26
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.19041
[debug] exe versions: none
[debug] Proxy map: {}
[pluralsight:course] google-cloud-platform-fundamentals-core-infrastructure: Downloading JSON metadata
[download] Downloading playlist: Google Cloud Platform Fundamentals - Core Infrastructure
[pluralsight:course] playlist Google Cloud Platform Fundamentals - Core Infrastructure: Collected 66 video ids (downloading 66 of them)
[download] Downloading video 1 of 66
[pluralsight] bb96b36c-3f87-4530-bf35-6cedcdfa9866-0: Downloading JSON metadata
[pluralsight] bb96b36c-3f87-4530-bf35-6cedcdfa9866-0: Downloading mp4-high-widescreen viewclip graphql
[pluralsight] bb96b36c-3f87-4530-bf35-6cedcdfa9866-0: Downloading mp4-high-widescreen viewclip JSON
[pluralsight] bb96b36c-3f87-4530-bf35-6cedcdfa9866-0: Waiting for 10 seconds to avoid throttling
ERROR: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpeav8wi2f\build\youtube_dl\YoutubeDL.py", line 806, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpeav8wi2f\build\youtube_dl\YoutubeDL.py", line 827, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpeav8wi2f\build\youtube_dl\extractor\common.py", line 534, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpeav8wi2f\build\youtube_dl\extractor\pluralsight.py", line 421, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpeav8wi2f\build\youtube_dl\extractor\common.py", line 1374, in _sort_formats
youtube_dl.utils.ExtractorError: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

## Description

 It said No video formats found

**Comment by cameronetchart** [comment]
I just ran into the same issue, but the issue is limited to youtube playlists for me (for me, my watch later playlist). When I downloaded the individual files using "-a" and removing the playlist "&list=" portion of the URL, they all download fine.

**Comment by cameronetchart** [comment]
@tuannv1104 I just had this issue, and tested this quite a bit. The bug is with using --cookies along with an expired cookie file. Delete your cookies.txt and recreate, then try again; that worked for me. Or, remove --cookies from your config file and that also works.

I've never had this happen before, but it also bugged out youtube-dlc for me as well, using the same cookies.txt file.

**Comment by tuannv1104** [comment]
@cameronetchart thanks for your help. I tried to remove --cookies from youtube-dl.conf and the error still happen. 

My config file is:
`--abort-on-error 
--no-warnings 
--console-title 
--batch-file='batch-file.txt' 
--user-agent "Mozilla/5.0 (X11; Linux i686; rv:21.0) Gecko/20100101 Firefox/21.0" 
--socket-timeout 10 
--retries 10 
--continue 
--fragment-retries 10 
--min-sleep-interval 60 
--max-sleep-interval 300 
--max-downloads 150 
-f 'best[height<=720]/worst[height>720]' 
-o '%(playlist_title)s/%(chapter_number)s - %(chapter)s/%(playlist_index)d-%(title)s.%(ext)s' `

**Comment by bitounu** [comment]
I have the very same error when I try to only get JSON info from video, I have "-s" option to skip downloading. My command looks like this:
youtube-dl --verbose --skip-download -j YOUTUBE_URL
I run it for every URL saved in archive file (around 30 movies). In consecutive 10 runs I got this error 1 or 2 times.
I will check if it could be a YT counteraction by making random sleeps between downloads and will report here.

**Comment by cameronetchart** [comment]
@tuannv1104 what I would try next is removing your config file entirely,
see if it works. If it does, start adding back your config entries until
you find the one that causes the error. That's what I had to do to find the
--cookies error. But if the downloads fail entirely even without any config
entries in youtube-dl.conf, then I'm out of ideas unfortunately.

I did this testing with the -F flag to make testing quicker.
